packages/website/src/components/inputs/Checkbox.tsx

```diff
@@ -10,17 +10,18 @@ export interface CheckboxProps {
 }
 
 function Checkbox(props: CheckboxProps): JSX.Element {
-  const checkboxRef = createRef<HTMLInputElement>();
+  const { indeterminate } = props;
 
-  useEffect(() => {
-    if (!checkboxRef.current) {
-      return;
-    }
+  const checkboxRef = useCallback(
```

Comment:
Small refactoring: Callback refs are generally the recommended way to interacting with the underlying DOM node:
https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node